### ISSUE TITLE
Deep Learning with Keras

### ISSUE THREAD

**Issue by Startonix** [Startonix/ModularAI-#155]
keras_nn.py

from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

model = Sequential()
model.add(Dense(64, activation='relu', input_dim=100))
model.add(Dense(10, activation='softmax'))

model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])

# Example data
import numpy as np

x_train = np.random.rand(1000, 100)
y_train = np.random.randint(10, size=1000)

model.fit(x_train, y_train, epochs=10, batch_size=32)
